website/docs/reference/manifest.md

```diff
@@ -320,6 +320,22 @@ By default, this flag is disabled.
 unstable-add-statements-functions-debug-info = true
 ```
 
+### `unstable-add-statements-code-locations-debug-info`
+
+> [!WARNING]
+> This is highly experimental and unstable feature intended to be used
+> by [cairo-coverage](https://github.com/software-mansion/cairo-coverage).
+> It may slow down the compilation - it is advised not to use it for other purposes than mentioned in
+> [cairo-coverage](https://github.com/software-mansion/cairo-coverage) documentation.
```

Comment:
nit: maybe we should link to those purposes exactly, so they are easier to discover?

Reply:
@ksew1 when should those be used with cairo-coverage? Always?